extras/bin/ideascube

```diff
@@ -17,4 +17,11 @@ export DEBUG=$DEBUG
 export STORAGE_ROOT=$STORAGE_ROOT
 export IDEASCUBE_ID=$IDEASCUBE_ID
 
+
+if [ "$1" == "start" ]; then
```

Comment:
Exactly, collectstatic is useless for development and migrate should really be run exclusively manually by the developer (especially if you're working on migrations themselves)

Is there any other use case than development for running locally with `manage.py`?

If not, then let's not do those automatically at all for this case.

Hooking in uwsgi.py is where I should have hooked in the first place. I know that, since this very pull-request is changing some of the uwsgi setup. But somehow, I still went with `ideascube start`. Don't ask me where that came from, I don't know.

I'll fix this.